docs/cloud-tutorials/database-management.md

```diff
@@ -60,19 +60,21 @@ To manage your applications' database schemas, you must define schema migrations
 DBOS Cloud is compatible with any schema management tool as long as all its dependencies and assets are stored in your application directory.
 We recommend using a Typescript-based migration tool like [Knex](https://knexjs.org/guide/migrations.html) or [TypeORM](https://typeorm.io/migrations).
 
-You configure your schema migrations in the `migrate` field of your [`dbos-config.yaml`](../api-reference/configuration.md).
+You configure your schema migrations in the `migrate` and `rollback` fields of your [`dbos-config.yaml`](../api-reference/configuration.md).
 You must supply a list of commands to run to migrate to your most recent schema version.
 For example, if you are using [Knex](https://knexjs.org/guide/migrations.html), you might use:
 
 ```yaml
 database:
   # Other fields omitted
   migrate: ['npx knex migrate:latest']
+  rollback: ['npx knex migrate:down']
 ```
 
-To run your migrations locally, run `npx dbos-sdk migrate`.
+To run your migrations locally, run `npx dbos-sdk migrate` or `npx dbos-sdk rollback`.
 
-When you [deploy](./application-management.md#registering-and-deploying-applications) an application to DBOS Cloud, it runs `npx dbos-sdk migrate` to apply all schema changes before starting your application.
+When you [deploy](./application-management.md#registering-and-deploying-applications) or [rollback](./application-management.md#rolling-back-applications) a DBOS Cloud application, it runs `npx dbos-sdk migrate` or `npx dbos-sdk rollback`, respectively, to apply all schema changes before starting your application.
+Note that database schema migration should be backward compatible with the schema used by your currently running workflows.
```

Comment:
This is a very subtle point that needs to be separated into an infobox and explained.  For example:

:::info
When you deploy a new version of an application, DBOS Cloud first runs schema migrations, then upgrades executors to use the new application version. Thus, you should be careful making breaking schema changes such as deleting or renaming a column that are not compatible with the previous application version.
:::